### PR TITLE
flume 1.6.0 (new formula)

### PR DESCRIPTION
**Pull request by forevergenin**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

### CODE REVIEW

File: Formula/flume.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+class Flume < Formula
+  desc "Hadoop-based distributed log collection and aggregation"
+  homepage "https://flume.apache.org"
+  url "https://www.apache.org/dyn/closer.cgi?path=flume/1.6.0/apache-flume-1.6.0-bin.tar.gz"
+  sha256 "0f7cef2f0128249893498a23401a0c8cb261e4516bc60f1885f8a3ae4475ed80"
+
+  bottle :unneeded
+
+  depends_on "hadoop"
+  depends_on :java
```
Comment (DomT4) [inline Formula/flume.rb]: `depends_on :java => "1.7+"` to match the `env_script` instructions?

File: Formula/flume.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+  def install
+    rm_f Dir["bin/*.cmd", "bin/*.ps1"]
+    libexec.install %w[conf docs lib tools]
+    bin.install Dir["bin/*"]
+    bin.env_script_all_files(libexec/"bin", Language::Java.java_home_env("1.7+"))
+  end
+
+  def caveats; <<-EOS.undent
+    Hadoop must be in your path for flume executable to work.
```
Comment (DomT4) [inline Formula/flume.rb]: With the hard `depends_on` this may be safe to remove.

File: Formula/flume.rb (hunk 1 above)
Comment (forevergenin) [inline Formula/flume.rb]: Added. Thanks !

File: Formula/flume.rb (hunk 2 above)
Comment (forevergenin) [inline Formula/flume.rb]: Removed.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-core/commit/7f29c1cb9f8dafbb8d036f3382f02a75fcb5dc82. Thank you for your contribution to Homebrew @forevergenin; we appreciate it! :smiley_cat:

**Pull request merged**